### PR TITLE
Create project

### PR DESCRIPTION
**Pull request by elkebab**
bilde

### CODE REVIEW

**Comment by garshol** [comment]
Institusjon og prosjektleder er obligatorisk og må søkes opp.

File: 26-register_registration/creator_navigates_to_description_tab.feature [hunk 1]
```diff
@@ -64,3 +65,24 @@ Feature: Creator navigates to Description tab
...
+      | Project manager          |
+      | Start date               |
+      | Internal reference       |
+    And they see a Cancel Button
+    And they see a Save Button
+
+  @xxx
+  Scenario: Creator creates a new Project
+    Given Creator opens Dialog for creating a new Project
+    When they enter a Project title
```
Comment (garshol) [inline 26-register_registration/creator_navigates_to_description_tab.feature:84]: And they look up a institution as Coordinating Institution
And ​they look up a person as Project Manager
And they set a Start date

**Review by garshol — DISMISSED** [review]
Add the three lines in comment

File: 26-register_registration/creator_navigates_to_description_tab.feature [hunk 2]
```diff
@@ -70,18 +70,21 @@ Feature: Creator navigates to Description tab
...
   Scenario: Creator creates a new Project
     Given Creator opens Dialog for creating a new Project
-    When they enter a Project title
+    When they enter a Project Title
+    And they select a Coordinating Institution
+    And ​they select a Project Manager
+    And they set a Start Date
     And they click Save
     Then the Dialog is closed
     And they see a confirmation message that the Project was created
```
Comment (garshol) [inline 26-register_registration/creator_navigates_to_description_tab.feature:90]: I’m unsure if this should be part of the same senario or documented separate, but the newly created project should also be added to the resource under registration.

Comment (elkebab) [reply 26-register_registration/creator_navigates_to_description_tab.feature:90]: Da erstatter jeg `And the Project can be found via Search` med `And they see the Project is added under Projects associations`

**Review by sarahse — APPROVED** [review]

**Review by garshol — APPROVED** [review]

**Pull request merged**